lib/Transforms/Scalar/LowerGvInitializers.cc

```diff
@@ -1,240 +1,242 @@
 #include "seahorn/Transforms/Scalar/LowerGvInitializers.hh"
 
-#include "boost/range.hpp"
 #include "boost/format.hpp"
+#include "boost/range.hpp"
```

Comment:
I should have probably split formatting changes and the fix to lowering. Is is better to leave it as-is, or make into 2 separate commits (the formatting one in a separate pull request)?

Reply:
2 separate commits so that it is clear what is your change and what is formatting would be nice